src/locales/de.json

```diff
@@ -1012,8 +1025,16 @@
         },
         "HeightmapTab": {
             "ColorSchemes": "Farbschemen",
+            "DefaultOrientation": "Standard Orientierung",
+            "DefaultOrientationDescription": "Standardausrichtung für die Darstellung der Höhenkarte auswählen.",
```

Comment:
Above you call it `Standard Orientierung` and here `Standardausrichtung`. It should be the same. `Standardausrichtung` sounds better imo